functions/src/model/ciBuilds.ts

```diff
@@ -76,10 +77,11 @@ export class CiBuilds {
     return snapshot.data() as CiBuild;
   };
 
-  public static getStartedBuilds = async (): Promise<CiBuild[]> => {
+  public static getStartedBuilds = async (limit = 30): Promise<CiBuild[]> => {
```

Comment:
Default parameter with a different value when being used. Should we get rid of the default param?